src/ImageBuilder.ts

```diff
@@ -100,19 +110,23 @@ export default class ImageBuilder {
             templateJson.properties.distribute[0].runOutputName = runOutputName;
             this.isVhdDistribute = templateJson.properties.distribute[0].type == "VHD";
 
-            var templateStr = JSON.stringify(templateJson);
+            var templateStr = JSON.stringify(templateJson, null, 2);
```

Comment:
What does the number 2 represent here? What do the options 0 and 1 do?

Reply:
here 2 is for the space
for 1  also its working fine but not with 0
when looked for it,most of the places found 2